server/views/pages/functionalSkills/index.njk

```diff
@@ -28,61 +28,114 @@
+        <span class="govuk-caption-l">Learning and work progress</span>
+        <h1 class="govuk-heading-l">{{ prisonerSummary.firstName | capitalize }} {{ prisonerSummary.lastName | capitalize }}'s functional skills</h1>
+
+        <div class="govuk-summary-card">
+          <div class="govuk-summary-card__title-wrapper">
+            <h2 class="govuk-summary-card__title">Current functional skills</h2>
+            {# Not MVP, Add assesment link
+            <ul class="govuk-summary-card__actions">
+                <li class="govuk-summary-card__action">
+                  <a class="govuk-link" href="">Update</a>
+                </li>
+            </ul>
+            #}
+          </div>
+          <div class="govuk-summary-card__content">
+            <table class="govuk-table" id="latest-functional-skills-table">
+              <caption class="govuk-table__caption govuk-visually-hidden">Details of current functional skills</caption>
+              <thead class="govuk-table__head">
+                <tr class="govuk-table__row">
+                  <th scope="col" class="govuk-table__header">Subject</th>
+                  <th scope="col" class="govuk-table__header">Assessed on</th>
+                  <th scope="col" class="govuk-table__header">Level</th>
+                  <th scope="col" class="govuk-table__header">Assessment type</th>
+                  <th scope="col" class="govuk-table__header"><span class="govuk-visually-hidden">Action</span></th>
+                </tr>
+              </thead>
+
+              <tbody class="govuk-table__body">
+              {% for assessment in latestFunctionalSkills.assessments %}
+                <tr class="govuk-table__row">
+                  <td class="govuk-table__cell">{{ assessment.type | formatFunctionalSkillType }}</td>
+                  {% if assessment.assessmentDate and assessment.grade %}
+                    <td class="govuk-table__cell">{{ assessment.assessmentDate | formatDate('D MMMM YYYY') }}</td>
+                    <td class="govuk-table__cell">{{ assessment.grade }}</td>
+                    <td class="govuk-table__cell">
+                      Initial
+                    </td>
+                    <td class="govuk-table__cell">
+                      <span class="govuk-hint">Curious</span>
+                    </td>
+                  {% else %}
+                    <td class="govuk-table__cell" colspan="4">No functional skills assessment recorded</td>
+                  {% endif %}
+                </tr>
+              {% endfor %}
+              </tbody>
+            </table>
+          </div>
         </div>
-        <div class="govuk-summary-card__content">
-          <table class="govuk-table" id="functional-skills-table">
-            <caption class="govuk-table__caption govuk-visually-hidden">Details of current functional skills</caption>
-            <thead class="govuk-table__head">
+
+        <div class="govuk-summary-card">
```

Comment:
This is the new table / summary card for the Assessment History (ie. all functional skills)